tool/gravity/cli/status.go

```diff
@@ -299,6 +302,31 @@ func printStatusText(cluster clusterStatus) {
 	}
 }
 
+func statusKapacitor(env *localenv.LocalEnvironment) error {
+	operator, err := env.SiteOperator()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	cluster, err := operator.GetLocalSite()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	alerts, err := statusapi.FetchKapacitorAlerts(*cluster)
+
+	_, _ = fmt.Fprintf(os.Stdout, "Cluster alerts:\n")
+
+	if len(alerts) != 0 {
+		printClusterAlerts(alerts, os.Stdout)
```

Comment:
I would consider exiting with a non-zero code to signify that there are alerts even though there maybe false positives. It might be helpful in automation.